wsgi/odyssy/basic/utils.py

```diff
@@ -18,3 +19,14 @@ def get_today_start():
 def get_today_end():
     """ set end as undefined date """
     return datetime.combine(datetime.today(), time.max)
+
+
+def paginate_view(request, query_set, num_items):
```

Comment:
Let's make the function behave like this - 
```python
def paginate_view(request, query_set, page=None, num_items=None):
    if page is None:
        page = request.GET.get('page', default=1)
    if num_items is None:
        num_items = request.GET.get('num_items', 10)
    ...
```

In this way, we will not have to parse `num_items` every time, `request` can do fine too.